pkgs/racket-doc/scribblings/reference/vectors.scrbl

```diff
@@ -47,7 +47,20 @@ all elements, so for mutable data, mutating an element will affect other element
   v
 ]
 
-This function takes time proportional to @racket[size].}
+This function takes time proportional to @racket[size].
+
+A common mistake is using @racket[make-vector] to create nested vectors.
+The following creates a vector that contain 3 vectors to the same vector.
```

Comment:
I think you meant "3 references to the same vector", not "3 vectors to the same vector".